source/test/F0.Minesweeper.Logic.Tests/DefaultLocationShufflerTest.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+using F0.Minesweeper.Logic.Abstractions;
+using F0.Minesweeper.Logic.LocationShuffler;
+using FluentAssertions;
+using Xunit;
+
+namespace F0.Minesweeper.Logic.Tests
+{
+	public class DefaultLocationShufflerTest
+	{
+		[Theory]
+		[InlineData(1)]
+		[InlineData(2)]
+		[InlineData(3)]
+		[InlineData(4)]
+		[InlineData(5)]
+		[InlineData(10)]
+		[InlineData(13)]
+		public void ShuffleAndTake_WithCount_DoesReturnCorrectCount(int count)
+		{
+			DefaultLocationShuffler locationShufflerUnderTest = new();
```

Comment:
we could avoid the cast in the next line with
`ILocationShuffler locationShufflerUnderTest = new DefaultLocationShuffler();`
instead

Reply:
Did that, but also with the other changes. Thanks!